pom.xml

```diff
@@ -161,29 +165,28 @@
             <plugin>
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
+                <version>${jacoco.version}</version>
```

Comment:
Version is already defined by parent. No need to repeat it here

Reply:
Removed.